frontend/src/stores/search.ts

```diff
@@ -208,6 +208,27 @@ export const useSearchStore = defineStore("search", {
     searchTypeIsSupported(state) {
       return isSearchTypeSupported(state.searchType)
     },
+    /**
+     * Returns the unique string representation of the current collection
+     * when making collection searches.
+     */
+    collectionValue(): null | string {
+      if (this.collectionParams === null) {
+        return null
+      }
+
+      switch (this.collectionParams.collection) {
+        case "creator": {
```

Comment:
It's been a while since I saw `switch`-`case` because the object access pattern `{}[]` appears so much more often. Nice!